client/src/pages/private/ParticipantView.js

```diff
@@ -30,7 +30,13 @@ export default () => {
             )}
           </CustomTabs>
         </Grid>
-        <Grid container alignItems='center' justify='flex-start' direction='column'>
+        <Grid
+          container
+          alignItems='center'
+          justify='flex-start'
+          direction='column'
+          style={{ overflowX: 'scroll' }}
```

Comment:
try to use class instead of inline-style

Reply:
That makes sense! I'll update that.